pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.3</version>
+        <version>1.644</version>
```

Comment:
Ok, current LTS [seems to be](https://jenkins-ci.org/changelog-stable/) 1.642.2. Will change it to that.